### ISSUE TITLE
Faster Release Cycle and Beta Branch for Quicker Updates

### ISSUE THREAD

**Issue by wommy** [futo-org/android-keyboard#549]
### Use Cases

a 2 week beta cycle and a 4 week stable release would do wonders

I feel like it's so so so close to being stable, but it still feels like it's in a beta

There's so many features that would absolutely bury gboard, but I find myself hopping between the two like just due to tiny minor inconvenient bugs

I feel like a little period of maybe 3 months 6 months with a little burst of like an elevated releases would make me never use gboard again,

often find myself checking over and over, fiending for an update

Thanks for doing everything you're doing 

I bet a bunch if people would sign up for a beta with even like a 3 week release tempo, which could help land feedback for a quicker main

Thanks again 

### Proposal

beta branch with faster updates

**Comment by Erron-26** [comment]
Yeah, I know the project is **pretty new**, and the dev might have been busy lately, but when you see the commits being pushed and notice that some of those annoying little bugs are getting fixed, it just makes you want that version even more. It might not be totally stable, but at least it would save me a few headaches

**Comment by NyxWinter** [comment]
There is this: https://keyboard.futo.org/nightly.apk

**Comment by wommy** [comment]
what's the process for requesting a refund 
@dean-futo

**Comment by abb128** [comment]
This is planned for when FUTO Store becomes ready as the main method of distribution, I'll leave this issue open
